src/sharedHelpers/dateAndTime.ts

```diff
@@ -215,7 +215,7 @@ function dateFnsLocale( i18nextLanguage: string ) {
   }
 }
 
-function formatISONoTimezone( date: Date ) {
+function formatISONoTimezone( date: Date | null | undefined ) {
```

Comment:
In `parseExif`, a `null | undefined` value is being passed here, which this function gracefully handles on the line below